helm/cloud-provider-vsphere/charts/vsphere-csi-driver/values.yaml

```diff
@@ -8,41 +8,6 @@ config:
   zone: ""
   thumbprint: ""
 
-controllerDeployment:
-  replicas: 1
-  csiAttacher:
```

Comment:
We use this chart in only cluster-vsphere and we prefer applying changes here instead of overwriting values in the upper layers. That is why I am OK with simplifying everything by accepting the maintenance cost (open PR here, create a new release, update the version in the upper layer etc) when we need to change just a value.

Reply:
Do you mean that I should revert this change to keep the capability to set image and tags ourselves?

Reply:
No no. I am OK with the change. Theoretically, we should have the configuration capability here but don't use it in upper layers. We always apply the changes here and then release a new version. That is why we can drop the configuration capability.

Reply:
I am now working on a way to string replace the upstream image registries to our own.

Reply:
It wasn't as straightforward as expected but it works. 
I suggest you take a look as this is post-review
https://github.com/giantswarm/cloud-provider-vsphere-app/pull/98/commits/a2deb56218fe94d86f9a2459722bf15dd5260c3b